README.md

```diff
@@ -1,2 +1,39 @@
 # triton-mysql
-MySQL designed for container-native deployment on Joyent's Triton platform
+
+MySQL designed for container-native deployment on Joyent's Triton platform.
+
+### Architecture
+
+We have a primary (`mysql_primary`) and replicas (`mysql`) as separate `docker-compose` services. The replicas ask Consul (via Containerbuddy) for the primary's IP and self-configure replication with the primary in their `onStart` handler. The replicas' `onChange` handler will move replication to a new master if one is created.
+
+
+### Configuration
+
+Pass these variables in your environment or via an .env file.
```

Comment:
Asked elsewhere: should it be a dot file?

Real question here: I typically backtick file names. Not sure if that's a formal style.